src/ray/stats/metric_defs.h

```diff
@@ -147,23 +218,37 @@ static Gauge RestoringBandwidthMB("object_restoration_bandwidth_mb",
 ///
```

Comment:
this comment could go away?

Reply:
I will remove this once we migrate all metrics to DEFINE_stats!